src/graphql/resolvers/index.ts

```diff
@@ -4,11 +4,15 @@ import { Resolvers } from '../_generated/types';
 import { query as Query } from './Query';
 import { jokes as Jokes } from './Jokes';
 import { joke as Joke } from './Joke';
+import { node as Node } from './Node';
+
+/** SWAPI resolvers */
+import swapiResolvers from './swapi';
 
 const resolvers: Resolvers = {
   Query,
   Jokes,
   Joke,
 };
 
-export default merge(resolvers);
+export default merge(resolvers, swapiResolvers, { Node });
```

Comment:
why not spread over `resolvers` ?
```typescript
const resolvers: Resolvers = {
 Query,
 Jokes,
 Joke,
...swapiResolvers,
{ Node }
};
```

Reply:
Generated TypeScript interface `Resolvers` does not contain SWAPI resolvers types.